src/scripts/towns/RoamerNPC.ts

```diff
@@ -17,7 +17,9 @@ class RoamerNPC extends NPC {
 
         // If no roaming Pokemon yet
         if (!roamers.length) {
-            return `There hasn't been any reports of roaming Pokémon around ${GameConstants.camelCaseToString(GameConstants.Region[this.region])} lately.`;
+            const regionName = RoamingPokemonList.roamerGroups[this.region]?.[this.subRegionRoamerGroup]?.name
+                ?? GameConstants.camelCaseToString(GameConstants.Region[this.region]);
+            return `There hasn't been any reports of roaming Pokémon around ${regionName} lately.`;
```

Comment:
```suggestion
            return `There haven't been any reports of roaming Pokémon around ${regionName} lately.`;
```
Might as well fix this while you're at it.